source/PreferencesPanel.cpp

```diff
@@ -178,6 +200,30 @@ bool PreferencesPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &comma
 		if(zones[latest].Value().KeyName() != Command::MENU.KeyName())
 			Command::SetKey(zones[latest].Value(), 0);
 	}
+	else if(key == 'i' && page == 'p')
+	{
+		page = 'i';
+		if(!downloadedInfo)
+		{
+			PluginHelper::Download(PLUGIN_LIST_URL, Files::Config() + "plugins.json");
```

Comment:
There should be some message somewhere if it fails to download this file.

Reply:
message created